pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication
```

Comment:
applications should pass in `python` or `pythonPackages`, not the individual attributes.